website/project/metadata/prereg-prize-test.json

```diff
@@ -178,51 +268,111 @@
+                "properties": {
+                    "question": {
+                        "id": "q17a",
+                        "type": "string",
+                        "format": "textarea"
+                    },
+                    "uploader": {
+                        "id": "q17b",
+                        "type": "osf-upload" 
+                    }
+                }
             },
             "q18": {
                 "nav": "Exclusion",
-                "type": "string",
-                "format": "textarea",
+                "type": "object",
                 "title": "How will you determine what data or samples (if any) to exclude from your analyses?",
                 "description": "Are there specific criteria that must be met in order to include? How will outliers be handled? You may want to attach a decision tree with an if, then” structure to easily formalize the way in which these decisions will be made.",
-                "help": "No checks will be performed to determine eligibility for inclusion besides verification that each subject answered each of the three tastiness indices. Outliers will be included in the analysis."
+                "help": "No checks will be performed to determine eligibility for inclusion besides verification that each subject answered each of the three tastiness indices. Outliers will be included in the analysis.",
+                "properties": {
+                    "question": {
+                        "id": "q18a",
+                        "type": "string",
+                        "format": "textarea"
+                    },
+                    "uploader": {
+                        "id": "q18b",
+                        "type": "osf-upload" 
+                    }
+                }
             },
             "q19": {
-                "nav": "Incoplete",
-                "type": "string",
-                "format": "textarea",
+                "nav": "Incomplete",
+                "type": "object",
                 "title": "How will you deal with incomplete or missing data?",
                 "description": "Almost every study will have missing data points. How you cope with these missing points is important because it can affect the result of your analysis.",
-                "help": "If any of the three indices of tastiness are not completed by a subject, that subject will not be included in the analysis."
+                "help": "If any of the three indices of tastiness are not completed by a subject, that subject will not be included in the analysis.",
+                "properties": {
+                    "question": {
+                        "id": "q19a",
+                        "type": "string",
+                        "format": "textarea"
+                    },
+                    "uploader": {
+                        "id": "q19b",
+                        "type": "osf-upload" 
```

Comment:
@leb2dg What do you think about adding a format field to the osf-upload type?

Then we could have a version where the filepicker is always visible (Script) and one that is toggleable for the optional uploads